### PR TITLE
Edit model to handle custom headers

### PR DESCRIPTION
**Pull request by danielpetrica**
Solving issue https://github.com/roblesterjr04/EloquentSalesForce/issues/99

I hope it's ok, I'm open to feedback if there are problems I haven't considered

### CODE REVIEW

File: src/Model.php [hunk 1]
```diff
@@ -217,9 +219,17 @@ protected function performUpdate($query)
             SObjects::authenticate();
             $object = $this->sfObject();
 
+            // The user can set this property on it's models to set some custom value for the headers
+            if (count($this->custom_headers) > 0) {
+                $headers = $this->custom_headers;
+            } else {
+                $headers = null;
+            }
+
```
Comment (Engveloper) [inline src/Model.php:228]: You can do this
```php
$headers = $this->custom_headers ?: null;
```

Comment (danielpetrica) [reply src/Model.php]: I applied the edits

**Comment by danielpetrica** [comment]
As I've updated the model can we merge it?

**Comment by danielpetrica** [comment]
Hi @roblesterjr04, sorry to ping but ca can we merge this pull request?

**Comment by roblesterjr04** [comment]
Sure, sorry for the tardiness!

**Comment by danielpetrica** [comment]
Thank you for the merge

**Pull request merged**